calico-enterprise/threat/suspicious-ips.mdx

```diff
@@ -125,17 +125,28 @@ Use the push method if your threat feeds that are not in newline-delimited forma
    kubectl apply -f <your_threatfeed_filename>
    ```
 
-3. Configure or program your threat feed to write updates to Elasticsearch. This Elasticsearch document is in the index **.tigera.ipset.\<cluster_name\>** and must have the ID set to the name of the global threat feed object. The doc should have a single field called **ips**, containing a list of IP prefixes. For example:
+3. Configure or program your threat feed to write updates to Elasticsearch. This Elasticsearch document is written using alias **tigera_secure_ee_threatfeeds_domainnameset.\<cluster_name\>.** and must have the ID set to the name of the global threat feed object. The doc should have a single field called **ips**, containing a list of IP prefixes. For example:
```

Comment:
This Elasticsearch document is written using the **tigera_secure_ee_threatfeeds_domainnameset.\<cluster_name\>.** alias and ...